src/plugins/maps_legacy/public/plugin.ts

```diff
@@ -26,17 +31,18 @@ import { getServiceSettings } from './get_service_settings';
 
 export const bindSetupCoreAndPlugins = (
```

Comment:
Don't forget to remove the method bindSetupCoreAndPlugins know that its no longer used anymore